### ISSUE TITLE
NVDA does not work after translating the following string

### ISSUE THREAD

**Issue by dingpengyu** [nvaccess/nvda#9342]
Hello everyone, I just tested svn's new nvda.po file and found that after translating the following string and replacing it in my test environment, NVDA won't start properly. Here is the specific string:
I understand that changing these settings may cause NVDA to function incorrectly. ID: 1937
Warning! 翻译 — 中文（中国）: 警告, ID: 1935
The following settings are for advanced users. Changing them may cause NVDA to function incorrectly. Please only change these if you know what you are doing or have been specifically instructed by NVDA developers. 翻译 — 中文（中国）: 警告！以下设置适用于高级用户。修改它们可能会导致NVDA无法正常运行。如果 您知道自己在做什么或者是已经得到NVDA开发人员的明确指示，才可以修改这些 设置。, ID: 1936
Restore defaults 翻译 — 中文（中国）: 用户默认, ID: 1938
NVDA Settings 翻译 — 中文（中国）: NVDA设置, ID: 1979

### Steps to reproduce:
First translate nvda.po and then generate nvda.mo, then overwrite nvda.mo to the translation directory, and finally restart NVDA.
### Actual behavior:
After restarting NVDA, there will be a system error tone and no error window will appear.
### Expected behavior:
Unable to start NVDA and read aloud
#### NVDA installed/portable/running from source:
install
#### NVDA version:
2019.1 beta
#### Windows version:
windows10-1809
#### Name and version of other software in use when reproducing the issue:
NVDA2019
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no
Error log provided by Russian
CRITICAL - __main__ (11:01:02.378):
core failure
Traceback (most recent call last):
File "nvda.pyw", line 233, in <module>
File "core.pyo", line 238, in main
File "appModuleHandler.pyo", line 26, in <module>
File "NVDAHelper.pyo", line 22, in <module>
File "eventHandler.pyo", line 9, in <module>
File "api.pyo", line 11, in <module>
File "review.pyo", line 10, in <module> 
File "NVDAObjects\__init__.pyo", line 17, in <module> 
File "displayModel.pyo", line 16, in <module> 
File "mouseHandler.pyo", line 8, in <module> 
File "gui\__init__.pyo", line 28, in <module> 
File "gui\settingsDialogs.pyo", line 2101, in <module> 
File "gui\settingsDialogs.pyo", line 2118, in AdvancedPanel 
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-8: ordinal not in range(128)

**Comment by josephsl** [comment]
Looks like regression from an attempt to tidy up the advanced panel. CC @feerrenrut

**Comment by josephsl** [comment]
Hi,

Note that this was reported to translations mailing list with another translator confirming the problem.

Thanks.

**Comment by zstanecic** [comment]
Hi Joseph and all,

I can confirm this problem for polish and Croatian languages, when copied the latest.mo file I translated to the alpha snap. 

The Unicode error is the same,

NVDA refuses to start. 

Best,

Zvonimir

**Comment by leonardder** [comment]
Fixes in #9342, not yet in master.
